langlib/langlib-test/src/test/resources/test-src/valuelib_test.bal

```diff
@@ -3662,6 +3662,7 @@ function testCloneWithTypeToTableNegative() {
 
     UnionTable|error t2 = tab.cloneWithType();
     assertTrue(t2 is error);
+    errMsg = "'table<map<anydata>>' value cannot be converted to 'UnionTable': " + errMsgSuffix;
```

Comment:
This is an existing [issue](#41288) that is fixed by this PR. Will include this in the release doc since this is a breaking change